Resources/Core/Build/Configuration/JSUnit/Bootstrap.js

```diff
@@ -23,11 +23,21 @@ var packages = [
 		location: '/base/typo3/sysext/core/Resources/Public/JavaScript/Contrib/lit-html',
 		main: 'lit-html'
 	},
+	{
+		name: '@lit/reactive-element',
+		location: '/base/typo3/sysext/core/Resources/Public/JavaScript/Contrib/@lit/reactive-element',
+		main: 'reactive-element'
+	},
 	{
 		name: 'lit-element',
 		location: '/base/typo3/sysext/core/Resources/Public/JavaScript/Contrib/lit-element',
 		main: 'lit-element'
```

Comment:
HEADS UP:
With `lit-element` v2 the `main` entry now actually is `index(.js)`.

We need to pick a decision here.
As we will not import from `lit-element` directly this currently doesn't matter for us.
For backwards compatibility for 3rd party extensions the core will use `index` here, but in testing framework we loose compatbility to 10.1/10.0 if we change this to `index`.

Therefore this pull requests stays with `lit-element` for now.

Reply:
I think that's fine. we should keep this in mind when making the next major version of the testing framework.